webapp/src/components/Header.js

```diff
@@ -1,38 +0,0 @@
-import sumaLogo from "../assets/images/suma-logo.png";
-import React from "react";
-import Nav from "react-bootstrap/Nav";
-import Navbar from "react-bootstrap/Navbar";
-
-const Header = ({ subText }) => {
```

Comment:
Keep components single-responsibility. This was doing nav, and the section header. Those are two different things. I got rid of the section header but kept the nav.